varinfo/variable.py

```diff
@@ -242,28 +242,23 @@ def _extract_references(self, attribute_string: str) -> set[str]:
 
     def _extract_dimensions(self, variable: ET.Element) -> list[str]:
         """Find the dimensions for the variable in question. If there are
-        overriding or supplemental dimensions from the CF configuration
-        file, these are used instead of, or in addition to, the raw
-        dimensions from the `.dmr`. All references are converted to
-        absolute paths in the granule. A set of all fully qualified
-        references is returned.
+        overriding dimensions from the `earthdata-varinfo` configuration
+        file, these are used instead of the raw dimensions from the `.dmr`. All
+        references are converted to absolute paths in the granule. A set of all
+        fully qualified references is returned.
 
         """
-        overrides = self.cf_config['cf_overrides'].get('dimensions')
-        supplements = self.cf_config['cf_supplements'].get('dimensions')
+        dimensions_override = self.cf_overrides.get('dimensions')
```

Comment:
When cutting out the `supplements` stuff, I thought `overrides` was a little too generic. I know the method gives context, but still the `overrides` to `dimensions_override` renaming is purely aesthetic.